docs/src/pages/[platform]/connected-components/storage/storageimage/react.mdx

```diff
@@ -43,15 +43,41 @@ At a minimum you must include the `alt`, `imgKey` and `accessLevel` props. `acce
   </ExampleCode>
 </Example>
 
+<Accordion.Container>
+  <Accordion.Item key={"title"} value={"title"}>
+    <Accordion.Trigger>
+      Using `@aws-amplify/ui-react-storage` version 3.0.1x or below?
+      <Accordion.Icon />
+    </Accordion.Trigger>
+    <Accordion.Content>
+      Versions 3.0.1x and earlier use `imgKey`, `accessLevel`, and `identityId` in place of the `path` prop. See [Amplify Gen 1 documentation](https://docs.amplify.aws/gen1/react/build-a-backend/storage/download/) for more information.
+      ```diff
+      <StorageImage 
+         alt="sleepy-cat" 
+      -  path="guest/cat.jpg"
+      +  accessLevel="guest" 
+      +  imgKey="cat.jpg" 
+      />
+
+       <StorageImage 
```

Comment:
Feels a little confusing to show a diff that goes backwards, think we could just the example as it would have worked in 3.0.18
```
like this instead? 
<StorageImage
   alt="sleepy-cat"
   accessLevel="private"
   identityId={identityId}
   imgKey="cat.jpg"
/>
```
Thoughts?

Reply:
Yeah that makes sense to me